run-all-tests.sh

```diff
@@ -0,0 +1,6 @@
+dirs=$(find collectors/ -type d -maxdepth 1)
```

Comment:
maybe we can have it in the Makefile?
And use `make -C collectors/okta`

Reply:
That sounds like a good solution. Though I'm struggling to find a way to list all the directories in the collectors directory with make.